test/integration/basic_nsmd_deploy_test.go

```diff
@@ -39,16 +41,35 @@ func testNSMgrDdataplaneDeploy(t *testing.T, nsmdPodFactory func(string, *v1.Nod
 	Expect(err).To(BeNil())
 	defer kubetest.ShowLogs(k8s, t)
 
-	nodes := k8s.GetNodesWait(2, defaultTimeout)
+	nodes := k8s.GetNodesWait(nodesCount, defaultTimeout)
 
-	if len(nodes) < 2 {
+	if len(nodes) < nodesCount {
 		logrus.Printf("At least two Kubernetes nodes are required for this test")
-		Expect(len(nodes)).To(Equal(2))
+		Expect(len(nodes)).To(Equal(nodesCount))
 		return
 	}
 
-	_, err = kubetest.SetupNodes(k8s, 2, defaultTimeout)
-	Expect(err).To(BeNil())
+	var wg sync.WaitGroup
+	for i := 0; i < nodesCount; i++ {
```

Comment:
Seems like nodes are independent in that test. Is there any point to do the same checks for more than one node?